tests/unit/s2n_async_pkey_test.c

```diff
@@ -26,6 +26,9 @@
 #include "utils/s2n_safety.h"
 #include "tls/s2n_async_pkey.h"
 
+/* Include source to test on a specific pkey_op type */
+#include "tls/s2n_async_pkey.c"
+
```

Comment:
This is no longer necessary! :D
https://github.com/aws/s2n-tls/blob/main/api/s2n.h#L624-L625

Reply:
Removed.